<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortChoose</class>
 <widget class="QWidget" name="SortChoose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>721</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="CheckBoxHeapSort">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>堆排序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="CheckBoxCountSort">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>计数排序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="CheckBoxBucketSort">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>桶排序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="CheckBoxQuickSort">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>快速排序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="CheckBoxHillSort">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>希尔排序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="CheckBoxBubbleSort">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>冒泡排序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="CheckBoxMergeSort">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>归并排序</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="LabelText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>选择要使用的算法：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnStart">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnClear">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ListWidgetTime">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="ListWidgetResult">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>491</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelTime">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>时间比较：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>排序结果：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
